HTML/index.html

```diff
@@ -1,18 +1,63 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Viren's Game</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <div class="title">
-      <h1>Viren's Game</h1>
-    </div>
-    <div class="screen">
-    
-    </div>
-  </body>
-  <script type="text/javascript" src="script.js">
+<head>
+  <title>Viren's Game</title>
+  <!-- Latest compiled and minified CSS -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
 
-  </script>
+  <!-- Optional theme -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+  <!-- Latest compiled and minified JavaScript -->
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+  <!-- My own css -->
+  <link rel="stylesheet" href="../css/indexStyle.css">
+</head>
+<!-- The navbar stretches across the width of the page -->
+<body class="container-fluid" onload="scoreBoard()">
+  <!-- First row for the title -->
+  <nav class="navbar-fluid navbar-inverse" data-offset-top="197">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="">VM</a>
+      </div>
+      <ul class="nav navbar-nav" style="float:left;">
+        <li class="hidden-xs hidden-sm"><a href="index.html">Home</a></li>
+        <li class="hidden-xs hidden-sm"><a href="Game1.html">Simon Says</a></li>
+        <li class="hidden-xs hidden-sm"><a href="Game2.html">Rock, Paper, Scissors</a></li>
+        <li class="hidden-xs hidden-sm"><a href="leaderBoard.html">LeaderBoard</a></li>
+        <li class="dropdown" style="width:200px;">
+          <a class="dropdown-toggle dropdown hidden-md hidden-lg" data-toggle="dropdown" href="#">
+            <span class="caret"></span>
+          </a>
+          <ul class="dropdown-menu">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="Game1.html">Simon Says</a></li>
+            <li><a href="Game2.html">Rock, Paper, Scissors</a></li>
+            <li><a href="leaderBoard.html" >LeaderBoard</a></li>
+          </ul>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  <header class="row">
+      <!-- Title of the page -->
+      <h1 class="col-xs-12 title">Viren's Game</h1>
+      <h2 class="col-xs-12 titleSub">Home Page</h2>
+  </header>
+  <!-- This holds images of the games -->
+  <div class="col-xs-8 col-xs-offset-2 display">
+    <div class="header"><h2>Available games</h2></div>
+    <a href="Game1.html"><div class="gameOneImage">
```

Comment:
this will work but is confusion break your different element tags onto new lines with nesting